test/_init_tests.js

```diff
@@ -160,6 +166,10 @@ class TestConfig {
     return this.if(this.platform.isIos.bind(this.platform));
   }
 
+  ifIe() {
+    return this.if(this.platform.isIe.bind(this.platform));
```

Comment:
That's correct, but we haven't had any platforms skipped by default yet. Check out the implementation of `this.if()`.